src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/Utils.cs

```diff
@@ -21,6 +21,9 @@ namespace OpenTelemetry.Contrib.Instrumentation.AWS.Implementation
 {
     internal class Utils
     {
+        internal static string ActivitySourceName =
```

Comment:
Looking at how this played out with Instrumentation.AwsLambda so far (https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/590#issuecomment-1219119832, #534) I think we should either:
* Rename the whole package instead of patching the activity source name (disruptive but consistent); or
* Leave everything as-is